integration_tests/e2e/workAndSkillsPage/workAndSkillsCoursesAndQualifications.cy.ts

```diff
@@ -18,44 +18,97 @@ context('Work and skills page - Courses And Qualification Card', () => {
       beforeEach(() => {
         cy.setupBannerStubs({ prisonerNumber })
         cy.setupWorkAndSkillsPageStubs({ prisonerNumber })
-        visitWorkAndSkillsPage()
       })
 
       context('Courses and Qualifications card', () => {
-        it('The card is displayed', () => {
-          const workAndSkillsPage = Page.verifyOnPage(WorkAndSkillsPage)
-          workAndSkillsPage.CAQ_card().should('exist')
-        })
-        it('The card summary header contains Courses and qualifications', () => {
+        it('should display course data given prisoner has at least one course completed in the last 12 months', () => {
```

Comment:
Cypress tests to assert the display of the card in each of the following scenarios as described in the jira ticket:

* Prisoner has at least completed course in the last 12 months
* Prisoner has completed courses, but none completed in the last 12 months
* Prisoner has attended courses, but has never completed any of them
* Prisoner has no course data at all
* Curious returns an error response when calling the API to get prisoner course data